features/step_definitions/sample_file_steps.rb

```diff
@@ -26,7 +26,7 @@ def a
 end
 
 Given(/^the "(.*?)" sample file exists$/) do |file_name|
-  full_path = File.expand_path file_name, 'spec/samples'
+  full_path = Pathname(__dir__) + '../../spec/samples' + file_name
```

Comment:
Hmmm. I really dislike the "+" for concatenation. Could we use string interpolation here and below? I believe thats also more idiomatic in regards to our code base.